pkg/rbeconfigsgen/rbeconfigsgen.go

```diff
@@ -496,25 +496,31 @@ func genCppConfigs(d *dockerRunner, o *Options, bazeliskPath string) (string, er
 	// 2. Harden each link.
 	// 3. Archive the contents of the config output directory into a tarball.
 	// 4. Copy the tarball from the container to the local temp directory.
-	out, err := d.execCmd("find", cppConfigDir, "-type", "l")
-	if err != nil {
-		return "", fmt.Errorf("unable to list symlinks in the C++ config generation build output directory: %w", err)
+	var out string
+	if o.ExecOS == "windows" {
+		out, err = d.execCmd("cmd", "/r", "dir", cppConfigDir, "/a:l", "/b")
+	} else {
+		out, err = d.execCmd("find", cppConfigDir, "-type", "l")
 	}
-	symlinks := strings.Split(out, "\n")
-	for _, s := range symlinks {
-		resolvedPath, err := d.execCmd("readlink", s)
-		if err != nil {
-			return "", fmt.Errorf("unable to determine what the symlink %q in %q in the toolchain container points to: %w", s, cppConfigDir, err)
-		}
-		if _, err := d.execCmd("ln", "-f", resolvedPath, s); err != nil {
-			return "", fmt.Errorf("failed to harden symlink %q in %q pointing to %q: %w", s, cppConfigDir, resolvedPath, err)
+	if err != nil {
```

Comment:
Can you check if find returns an error if symlinks aren't found? If the command behaves similar on both platforms I suggest removing `WARNING` here and just saying "no symlinks were found in the Bazel output directory that needed hardening. Ignoring the error returned by the symlinks find command".

If the error is specific to windows, then we should only ignore the error on windows.